les/downloader/statesync.go

```diff
@@ -610,6 +609,6 @@ func (s *stateSync) updateStats(written, duplicate, unexpected int, duration tim
 		log.Info("Imported new state entries", "count", written, "elapsed", common.PrettyDuration(duration), "processed", s.d.syncStatsState.processed, "pending", s.d.syncStatsState.pending, "trieretry", len(s.trieTasks), "coderetry", len(s.codeTasks), "duplicate", s.d.syncStatsState.duplicate, "unexpected", s.d.syncStatsState.unexpected)
 	}
 	if written > 0 {
-		rawdb.WriteFastTrieProgress(s.d.stateDB, s.d.syncStatsState.processed)
+		//rawdb.WriteFastTrieProgress(s.d.stateDB, s.d.syncStatsState.processed)
 	}
```

Comment:
Either the clause can be removed entirely or this should be uncommented again?

Reply:
There's no fast sync in `les`, so either way it doesn't matter. I think I just needed to remove references to the main `downloader.SyncProcess` types, which have no notion of fast sync any more. I could have rewritten `les`'s downloader too, but figured that it's gonna happen anyway from scratch so better just keep the changes minimal and don't try to make actual "real" changes. I guess I left as a comment to keep te original code intact if we want to see what was there.